README.md

```diff
@@ -13,25 +13,13 @@ For example under Debian Linux
 sudo apt install docker.io
 ```
 
-Then build the example image
+Then open the menu to select the desired image and options:
 
 ```shell
-./kas-container build kas-iot2050-example.yml
+./kas-container menu
```

Comment:
Doesn't work for my tmux configuration:

```shell
❯ ./kas-container menu
2022-01-07 08:18:42 - INFO     - kas 2.6 started
Unknown terminal: tmux-256color
Check the TERM environment variable.
Also make sure that the terminal is defined in the terminfo database.
Alternatively, set the TERMCAP environment variable to the desired
termcap entry.
```

Reply:
What's your `TERM` on the console side?

Reply:
Probably a kas-container topic: We may only propagate supported `TERM` values, similar to `SHELL`. Use a proper shell env, and it will work.

Reply:
❯ echo $TERM
tmux-256color

Reply:
And that is not ncurses-compatible. Check `/lib/terminfo` for supported settings. Not sure if Debian has an extension for tmux as well somewhere, maybe not.

Reply:
Buster vs. bullseye topic: ncurses from the latter addresses this. Will be fixed by kas 3.0 (once released) and by us picking up that version eventually.

Reply:
> Probably a kas-container topic: We may only propagate supported `TERM` values, similar to `SHELL`. Use a proper shell env, and it will work.

Yep, another zsh shell works well.